libcontainer/apparmor/apparmor_linux.go

```diff
@@ -1,22 +1,29 @@
 package apparmor
 
 import (
-	"bytes"
 	"errors"
 	"fmt"
 	"io/ioutil"
 	"os"
+	"sync"
 
 	"github.com/opencontainers/runc/libcontainer/utils"
 )
 
+var (
+	appArmorSupported bool
```

Comment:
nit: `s/Supported/Enabled/`

Reply:
Dang! I looked at it and wanted to update, but forgot: done 👍